### DISCUSSION TITLE
Does NeuralProphet implement training models across multiple time series?

### DISCUSSION THREAD

**Discussion by kgolyaev** [ourownstory/neural_prophet#208] [Q&A - get help using NeuralProphet] [answered]
I like the idea for the package quite a bit. I am wondering if currently it works the same way as base Prophet, which must be trained separately for each time series? And if this is the case, do you have it on your roadmap to implement support for multi-series model training?

**Comment by ourownstory** [comment, marked as answer]
Hi @kgolyaev 
> I am wondering if currently it works the same way as base Prophet, which must be trained separately for each time series?

Yes

> And if this is the case, do you have it on your roadmap to implement support for multi-series model training?

Yes, this should come soon, tracked in issue #103
